src/controllers/rankings.js

```diff
@@ -0,0 +1,66 @@
+const { graphql } = require('graphql');
+const { schema } = require('./graphql')
+
+const query = `
+query UserRankings($user_id: Int) {
+  userRankings(prev_ranking_idIsNull: true, user_id: $user_id) {
+    ...RankingFields
+    ...RankingsRecursive
+  }
+}
+
+fragment RankingsRecursive on UserRankings {
+  nextRanking {
+    ...RankingFields
+    nextRanking {
+      ...RankingFields
+      nextRanking {
+        ...RankingFields
+        nextRanking {
+          ...RankingFields
+          nextRanking {
+            ...RankingFields
+            nextRanking {
+              ...RankingFields
+              nextRanking {
+                ...RankingFields
+                nextRanking {
+                  ...RankingFields
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+fragment RankingFields on UserRankings {
+  ranking_id: id
+  item {
+    id
+    item_name
+    item_details {
+      description
+    }
+  }
+}
```

Comment:
Maybe easier / better / clearer to just get all rankings for user and do this in javascript

Reply:
I'm basically doing this with objection eager loading to get the string of next->next->next.

Reply:
Though, it's kinda a lame way for getting the order of lists without a max size limit.